javascript-bank/bank.js

```diff
@@ -0,0 +1,33 @@
+/* exported Bank */
+function Bank() {
+this.nextAccountNumber = 1;
+this.accounts = [];
+};
+
+Bank.prototype.openAccount = function(holder, balance) {
+  if (balance > 0) {
+   var obj = new Account(this.nextAccountNumber, holder);
+   obj.deposit(balance);
+   this.accounts.push(obj);
+   this.nextAccountNumber++;
+   return this.nextAccountNumber - 1;
+  } else {
```

Comment:
Please fix the `Bank` constructor's & `openAccount` method's indenting to be consistent with the rest of the code base (2 spaces per level of indenting).